Logger returns 2 values

# github.com/yeqown/fasthttp-reverse-proxy
/home/abhijit/go/src/github.com/yeqown/fasthttp-reverse-proxy/debug.go:9:2: assignment mismatch: 1 variable but "github.com/yeqown/log".NewLogger returns 2 values

--- a/debug.go
+++ b/debug.go
@@ -5,8 +5,8 @@ import (
 )
 
 var (
-	debug  bool = true
-	logger      = log.NewLogger()
+	debug     = true
+	logger, _ = log.NewLogger()
 )
 
 // SetProduction .

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ require (
 	github.com/fasthttp/websocket v1.4.1
 	github.com/imdario/mergo v0.3.9
 	github.com/valyala/fasthttp v1.4.0
-	github.com/yeqown/log v0.0.0-20200108034421-d68941cd8fd3
+	github.com/yeqown/log v1.0.3
 )
 
 go 1.13

